scripts/start_aerospike_in_docker.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-docker run -d --name aerospike -e "NAMESPACE=test" -p 3000-3002:3000-3002 aerospike:ce-5.7.0.8
\ No newline at end of file
+docker run -d --name aerospike -e "NAMESPACE=test" -p 3000-3002:3000-3002 aerospike:ce-6.2.0.7
```

Comment:
Added this in in order to help folks (like me) get the latest image, at least for now.  Maybe something like aerospike:latest is available to use?

Reply:
My preference is that we are always explicit with the version here. That said, we should keep this in sync with what the underlying client supports.

Reply:
Cool cool!